main/src/domain/targets/track_peak_target.rs

```diff
@@ -60,9 +60,17 @@ impl<'a> Target<'a> for TrackPeakTarget {
 impl TrackPeakTarget {
     fn peak(&self) -> Option<Volume> {
         let reaper = Reaper::get().medium_reaper();
-        let channel_count = unsafe {
-            reaper.get_media_track_info_value(self.track.raw(), TrackAttributeKey::Nchan) as i32
+        
```

Comment:
Nitpick: In order to comply with existing code, it would be better to remove the empty lines (I usually don't use empty lines to separate code regions). Thanks!